http-client-core/src/main/java/io/micronaut/http/client/HttpClientConfiguration.java

```diff
@@ -872,6 +875,28 @@ public HttpClientConfiguration.Http2ClientConfiguration getHttp2Configuration()
         return null;
     }
 
+    /**
+     * The path pattern to use for logging outgoing connections to pcap. This is an unsupported option: Behavior may
+     * change, or it may disappear entirely, without notice! Only implemented for netty.
+     *
+     * @return The path pattern, or {@code null} if logging is disabled.
+     */
+    @Internal
+    public String getPcapLoggingPathPattern() {
```

Comment:
What's the point of having the accessors internal? People will not see the property configuration as an internal

Reply:
the comment is for config users, the annotation is for programmatic users.